removed excess notes from "Change the Font Size of an Element" challe…

tadake opened the pull request:
<!-- FreeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->

<!-- Help with PRs can be found at https://gitter.im/FreeCodeCamp/Contributors -->

<!-- Make sure that your PR is not a duplicate -->
#### Pre-Submission Checklist

<!-- Go over all points below, and after creating the PR, tick all the checkboxes that apply. -->

<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->

<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

<!-- What type of change does your code introduce? After creating the PR, tick the checkboxes that apply. -->
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

<!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->

<!-- If you're unsure about any of these, don't hesitate to ask in the Help Contributors room linked above. We're here to help! -->
- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #11153 
#### Description

<!-- Describe your changes in detail -->

```
removed "Note Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge." text from "Change the Font Size of an Element" challenge. Other translations were fixed by using google translate.
```

Commit:
Merge pull request #10526 from FreeCodeCamp/staging

Release staging

Commit:
Merge pull request #10530 from FreeCodeCamp/staging

Release staging

Commit:
Merge pull request #10547 from FreeCodeCamp/staging

Release staging

Commit:
Merge pull request #10604 from FreeCodeCamp/staging

Release staging

Commit:
Merge pull request #10654 from FreeCodeCamp/staging

Release staging

tadake committed:
removed excess notes from "Change the Font Size of an Element" challenge.

removed "Note Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge." text from "Change the Font Size of an Element" challenge.

tadake committed:
removed excess notes from "Change the Font Size of an Element" challenge.

removed "Note Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge." text from "Change the Font Size of an Element" challenge.

Removed outdated warning.

removed " Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge." text from "Change the Font Size of an Element" challenge. Other translations were fixed by using google translate.

systimotic commented on seed/challenges/01-front-end-development-certification/html5-and-css.json:
```diff
@@ -790,8 +790,7 @@
         "Inside the same <code>&#60;style&#62;</code> tag that contains your <code>red-text</code> class, create an entry for <code>p</code> elements and set the <code>font-size</code> to 16 pixels (<code>16px</code>).",
-        "<strong>Note</strong><br>Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge.",
-        "Also, please do not add a class attribute to your new <code>p</code> element."
+        "<strong>Note</strong><br> please do not add a class attribute to your new <code>p</code> element."
```
Could you please capitalize this sentence?

systimotic commented on seed/challenges/01-front-end-development-certification/html5-and-css.json:
```diff
@@ -832,8 +830,7 @@
             "À l'intérieur de la même balise <code>&#60;style&#62;</code> qui contiens votre classe <code>.red-text</code>, créez une nouvelle entrée pour les éléments <code>p</code> et paramétrer le <code>font-size</code> à 16 pixels (<code>16px</code>).",
-            "<strong>Prenez note</strong><br>Dû aux différences entre les navigateurs Web, votre niveau de zoom devrait être à 100% pour passer les tests de ce défi.",
-            "Également, veuillez ne pas ajouter d'attributs de classe à votre nouvel élément <code>p</code>."
+            "<strong>Prenez note</strong><br> veuillez ne pas ajouter d'attributs de classe à votre nouvel élément <code>p</code>."
```
Could you capitalize this as well?

systimotic commented on seed/challenges/01-front-end-development-certification/html5-and-css.json:
```diff
@@ -854,8 +851,7 @@
             "Внутри того же элемента <code>&#60;style&#62;</code>, который содержит ваш класс <code>red-text</code>, создайте вхождение для элементов <code>p</code> и установите свойство <code>font-size</code> равным 16 пикселей (<code>16px</code>).",
-            "<strong>Внимание</strong><br>По причине разницы в реализации браузеров, вам может понадобиться установить 100% масштаб окна браузера для прохождения этого испытания.",
-            "Также, пожалуйста, не добавляйте атрибут класс к вашему новому элементу <code>p</code>."
+            "<strong>Внимание</strong><br> пожалуйста, не добавляйте атрибут класс к вашему новому элементу <code>p</code>."
```
Capitalization of this is a bit more complicated, because I don't understand this alphabet (I do speak limited German and French). Google Translate tells me it's `П` instead of `п`. I'll try to consult my Russian-speaking friends about this one.

systimotic replied:
They just replied saying that that's indeed what it should be

systimotic reviewed (changes requested):
I love that you picked this up, nice work!

tadake commented:
oops Xp. I'll capitalize all of that later today. thanks for pointing that out! @systimotic

systimotic commented:
No problem, I'm glad you're on top of it!

tadake committed:
Removed outdated warning.

removed " Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge." text from "Change the Font Size of an Element" challenge. Other translations were fixed by using google translate.

tadake commented:
I capitalized what was needed @systimotic :+1:

tadake committed:
Merge branch 'master' of https://github.com/tadake/FreeCodeCamp into fix/remove-excess-notes

tadake committed:
Merge branch 'fix/remove-excess-notes' of https://github.com/tadake/FreeCodeCamp into fix/remove-excess-notes

tadake committed:
removed "Note Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge." text from "Change the Font Size of an Element" challenge. Other translations were fixed by using google translate.

tadake committed:
Merge branch 'fix/remove-excess-notes' of https://github.com/tadake/FreeCodeCamp into fix/remove-excess-notes
